Bottom: 1d02fc67b57a8e8082d0fc05194919d141d0cb1f
Top:    677c27b6a5235a8a363b63ad454dc0928d773761
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-08 10:31:51 +0200

Refresh of update-clang-format-settings

---

diff --git a/.clang-format b/.clang-format
index 1a2d270..d29ff26 100644
--- a/.clang-format
+++ b/.clang-format
@@ -1,3 +1,3 @@
 Language:      Cpp
 BasedOnStyle:  LLVM
-
+Standard:      c++20
